<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LCDWidget</class>
 <widget class="QWidget" name="LCDWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>182</width>
    <height>96</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>182</width>
    <height>96</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="title_1">
       <property name="text">
        <string>Axe X</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="LCD_1">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Ignored">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>80</width>
         <height>20</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(0, 0, 0);
color: rgb(0, 255, 0);</string>
       </property>
       <property name="text">
        <string>-        635.658</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="unit_1">
       <property name="text">
        <string>mm</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QWidget" name="endLED_1" native="true">
       <property name="minimumSize">
        <size>
         <width>20</width>
         <height>10</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>20</width>
         <height>10</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(0,255, 0);
</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="title_2">
       <property name="text">
        <string>Axe Y</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="LCD_2">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Ignored">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>80</width>
         <height>20</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(0, 0, 0);
color: rgb(0, 255, 0);</string>
       </property>
       <property name="text">
        <string>+           0.000</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="unit_2">
       <property name="text">
        <string>mm</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QWidget" name="endLED_2" native="true">
       <property name="minimumSize">
        <size>
         <width>20</width>
         <height>10</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>20</width>
         <height>10</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(55, 0, 0);</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="title_3">
       <property name="text">
        <string>Axe Z</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="LCD_3">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Ignored">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>80</width>
         <height>20</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(0, 0, 0);
color: rgb(0, 255, 0);</string>
       </property>
       <property name="text">
        <string>-          12.564</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="unit_3">
       <property name="text">
        <string>mm</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QWidget" name="endLED_3" native="true">
       <property name="minimumSize">
        <size>
         <width>20</width>
         <height>10</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>20</width>
         <height>10</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(55, 0, 0);
border-color: rgb(0, 0, 0);</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
